src/igvfd/schemas/analysis_set.json

```diff
@@ -38,28 +38,56 @@
     "dependentSchemas": {
         "$merge": [
             "file_set.json#/dependentSchemas"
-        ]
+        ],
+        "file_set_type": {
```

Comment:
I think you need to add file_set_type to required properties

Reply:
fixed